### PR TITLE
fix:  Monitors -> Uptime resovles #1333

### PR DESCRIPTION
**Pull request by ajhollid**
This PR renames "Monitors" to Uptime and updates all references

- [x] Change "Monitors" to "Uptime"
- [x] Change all links and references "monitors" -> "uptime"

### CODE REVIEW

File: Client/src/Pages/Uptime/Details/index.jsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ const DetailsPage = ({ isAdmin }) => {
 			}
 		};
 		fetchCertificate();
-	}, [authToken, monitorId, monitor]);
+	}, [authToken, monitorId, monitor, dateRange, uiTimezone, dateFormat]);
```
Comment (marcelluscaio) [inline Client/src/Pages/Uptime/Details/index.jsx:103]: Why did we add these 3 dependencies?

**Review by marcelluscaio — COMMENTED** [review]
Great job. Just one question there

File: Client/src/Pages/Uptime/Details/index.jsx (hunk 1 above)
Comment (ajhollid) [reply Client/src/Pages/Uptime/Details/index.jsx:103]: I believe they are used in the hook, so we probably want the hook to run again if those values change?

Comment (marcelluscaio) [reply Client/src/Pages/Uptime/Details/index.jsx:103]: Oh yeah, I found two of them but not dateRange

Comment (ajhollid) [reply Client/src/Pages/Uptime/Details/index.jsx]: > Oh yeah, I found two of them but not dateRange

You're right, good catch!  I must have fat-fingered it in by accident.  Removed and commited, thank you!

**Review by marcelluscaio — APPROVED** [review]

**Pull request merged**